www/include/core/footer/footerPart.php

```diff
@@ -181,11 +181,11 @@ function initWholePage() {
  * set quick search position
  */
 function setQuickSearchPosition() {
-    if (jQuery('QuickSearch')) {
-        if (jQuery('header').is(':visible')) {
-            jQuery('QuickSearch').css({ top: '86px' });
+    if ($('QuickSearch')) {
```

Comment:
Please don't change jQuery to $.
Because is created an error JS on centreon module.
Thanks :) 

ps : on this function